osquery/core/tests/process_tests.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+ 
+#include <gtest/gtest.h>
+
+#include <osquery/core.h>
+
+#include "osquery/core/process.h"
+#include "osquery/core/test_util.h"
+
+extern char *self_exec_path;
+
+bool getProcessExitCode(osquery::PlatformProcess& process, int& exitCode)
```

Comment:
Isn't this function only relevant to this test? Maybe just declare it `static` since it is only relevant to this file?